backend/api/controllers/PullRequestController.js

```diff
@@ -1,20 +1,56 @@
 const express = require('express')
 const PullRequest = require('../../services/PullRequest')
+const redisClient = require('../../redis')
 
 const PullRequestController = express()
 
 PullRequestController.get('/:year?', async function (req, res) {
   let response = {}
-  let pullRequest = new PullRequest(req.params.year)
+  let year = req.params.year
```

Comment:
- There is no fallback for year
- must be a const, never updated

```js
const { year = new Date().getFullYear() } = req.params
```

Reply:
Quite cool syntax!

Reply:
I am not sure it works tho

Reply:
It does, since Node 6.4 https://node.green/#ES2015-syntax-destructuring--assignment-with-objects :)

Reply:
Could you please give me a link to its doc somewhere?